src/main/java/com/dnastack/wes/files/RunFileController.java

```diff
@@ -0,0 +1,62 @@
+package com.dnastack.wes.files;
+
+import com.dnastack.audit.aspect.AuditActionUri;
+import com.dnastack.wes.utils.RangeHeaderUtils;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@Slf4j
+@RestController
+@RequestMapping("/ga4gh/wes/v1")
+public class RunFileController {
+
+
+    private final RunFileService runFileService;
+
+    public RunFileController(RunFileService runFileService) {this.runFileService = runFileService;}
+
+    @AuditActionUri("wes:run:files:list")
```

Comment:
I always forget what is the standard here, shouldn't it be `wes:runs:files:list`? most right is an action, rest is a resource. Similar to REST API.